inst/sql/sql_server/OptimizeConceptSet.sql

```diff
@@ -0,0 +1,195 @@
+DROP TABLE IF EXISTS #o_not_excluded;
```

Comment:
Again, drop table if exists is postgres specific syntax. This will break most for most platforms